certified-connectors/AIForged/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4627 @@
+        },
+        "type": {
+          "$ref": "#/definitions/AIForged.DAL.SettingType"
+        },
+        "status": {
+          "$ref": "#/definitions/AIForged.DAL.SettingStatus"
+        },
+        "dtc": {
+          "type": "string",
+          "format": "date-time"
+        },
+        "dtm": {
+          "type": "string",
+          "format": "date-time"
+        },
+        "comment": {
+          "type": "string"
+        },
+        "info": {
+          "type": "string"
+        },
+        "data": {
+          "type": "string"
+        },
+        "minValue": {
+          "type": "string"
+        },
+        "maxValue": {
+          "type": "string"
+        },
+        "confidence": {
+          "type": "number",
+          "format": "float"
+        },
+        "minConfidence": {
+          "type": "number",
+          "format": "float"
+        },
+        "maxConfidence": {
+          "type": "number",
+          "format": "float"
+        },
+        "isCaseSensative": {
+          "type": "boolean"
+        },
+        "invert": {
+          "type": "boolean"
+        },
+        "oneLine": {
+          "type": "boolean"
+        },
+        "oneWord": {
+          "type": "boolean"
+        },
+        "isHandwriting": {
+          "type": "boolean"
+        },
+        "orientation": {
+          "$ref": "#/definitions/AIForged.DAL.Orientation"
+        },
+        "marking": {
+          "$ref": "#/definitions/AIForged.DAL.MarkingType"
+        },
+        "cells": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "clearBefore": {
+          "$ref": "#/definitions/AIForged.DAL.OptionStatusFlags"
+        },
+        "clearAfter": {
+          "$ref": "#/definitions/AIForged.DAL.OptionStatusFlags"
+        },
+        "cleanupValuesBefore": {
+          "type": "boolean"
+        },
+        "cleanupValuesAfter": {
+          "type": "boolean"
+        },
+        "validateValuesBefore": {
+          "type": "boolean"
+        },
+        "validateValuesAfter": {
+          "type": "boolean"
+        },
+        "abortOnValidationError": {
+          "type": "boolean"
+        },
+        "isReplacementCaseSensative": {
+          "type": "boolean"
+        },
+        "compileResults": {
+          "type": "string"
+        },
+        "maxRetry": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "timeout": {
+          "type": "string"
+        },
+        "userId": {
+          "type": "string"
+        }
+      }
+    },
+    "AIForged.DAL.SettingType": {
+      "type": "integer",
+      "description": "",
+      "x-enumNames": [
+        "None",
+        "Setting",
+        "Rule",
+        "Dependency",
+        "Template",
+        "Runtime"
+      ],
+      "enum": [
+        0,
+        1,
+        2,
+        3,
+        9,
+        10
+      ]
+    },
+    "AIForged.DAL.SettingStatus": {
+      "type": "integer",
+      "description": "",
```

Comment:
@aiforged Empty property values can be removed until unless you have a specific reason to keep them.

Applies the same to all other similar empty property values.